Pipfile

```diff
@@ -7,21 +7,17 @@ name = "pypi"
 click = "*"
 xarray = "*"
 Flask = "*"
-netCDF4 = "*"
+netCDF4 = "==1.5.6"
 gunicorn = "*"
+requests = "*"
+dask = {extras = [ "dataframe",]}
+e1839a8 = {path = ".", editable = true}
```

Comment:
The closest I was able to do was add `pipenv run pip install -e .` to the [python-ci](https://github.com/pacificclimate/orca/blob/i34-metadata-requests/.github/workflows/python-ci.yml#L34) and the [Makefile](https://github.com/pacificclimate/orca/blob/i34-metadata-requests/Makefile#L6) to ensure GitHub Actions and users installing with the Makefile can properly import the package when running automated tests. I don't imagine this is what you had in mind since I'm still using `pipenv`, so I was wondering if there's a way I could fix that issue without using `pipenv`.

Reply:
That's exactly what I had in mind. Thanks!